plugins/nexus-repository-manager/src/components/NexusRepositoryManager/NexusRepositoryManager.tsx

```diff
@@ -56,29 +36,35 @@ export function NexusRepositoryManager() {
     );
   }
 
-  const data = components?.map(v => {
+  const artifacts = components?.map(v => {
     const { component } = v;
 
     // theres only one asset per docker.image-name component
     // if we want to support multiple repository types
     // this will probably need to change in the future,
-    const asset = component.assets?.at(0);
+    const firstAsset = component.assets?.at(0);
+
+    // Artifact types that we want to display: either classifiers (e.g. javadoc) or extensions (e.g. zip)
+    const assetVariants = component.assets?.flatMap(asset => {
+      if (!asset.maven2) return [];
 
-    const shortHash = getSha256(asset)?.substring(0, 12);
-    const size = getFileSize(v);
+      const { classifier, extension } = asset.maven2;
+      if (extension === 'jar' && classifier) return classifier;
+      if (!hasIgnoredExtension(asset) && extension) return extension;
+      return [];
+    });
 
     return {
       version: component.version,
-      repository: component.name,
+      // TODO should we include maven groupID
+      artifact: component.name,
+      assetVariants: assetVariants,
       repositoryType: component.repository,
-      manifestDigest: (
-        <Box sx={{ display: 'flex', alignItems: 'center' }}>
-          <Chip label="sha256" className={classes.chip} />
-          {shortHash}
-        </Box>
-      ),
-      lastModified: formatDate(asset?.lastModified),
-      size: formatByteSize(size),
+      hash: getHash(firstAsset),
+      lastModified: formatDate(firstAsset?.lastModified),
+      // TODO for a maven component that uploads e.g. a protobuf, jar, and zip,
```

Comment:
This also does not gracefully handle the case where ALL assets are classified; I don't know if this ever happens.

Reply:
I think standard jar is a good default choice.

In the future there could be a request to support a glob pattern so folks can see the size of their `*-with-dependencies.jar` artifacts, which some runtimes require to avoid dependency clashing issues.

Reply:
I think we have to sum the sizes -- what about repos which don't publish a jar?